src/views/home/components/header/header.styles.ts

```diff
@@ -0,0 +1,65 @@
+import { createUseStyles } from "react-jss";
+
+import { Theme } from "src/styles/theme";
+
+const flexBox = { width: "100%", display: "flex" };
+const btn = (theme: Theme) => ({
+  display: "inline-flex",
+  alignItems: "center",
+  padding: [[theme.spacing(0.75), theme.spacing(1)]],
+  borderRadius: theme.spacing(1),
+  height: theme.spacing(4.5),
+  "& svg": {
+    width: 24,
+  },
+  "& p": {
+    margin: [[0, theme.spacing(1)]],
+    display: "none",
+    [theme.breakpoints.upSm]: {
+      display: "inherit",
+    },
+  },
+  "&:hover": {
+    backgroundColor: theme.palette.grey.light,
+  },
+});
+
+const useHeaderHomeStyles = createUseStyles((theme: Theme) => ({
+  header: {
+    ...flexBox,
+    justifyContent: "space-between",
+    fontWeight: theme.fontWeights.medium,
+    "& > *": {
+      width: "100%",
+    },
+  },
+  settingsBtn: {
+    ...flexBox,
+    alignItems: "center",
+    "& a": {
+      ...btn(theme),
+    },
+  },
+  myAccountBtn: {
+    ...flexBox,
```

Comment:
Given that these buttons are in a flex parent, instead of the `width: "100%"` I'd use `flex: 1`.